quisp/modules/Application.cc

```diff
@@ -60,185 +24,174 @@ Application::Application() { generatePacket = nullptr; }
   if (!toRouterGate->isConnected()) {
-    // Since we only need this module in EndNode, delete it otherwise.
     deleteThisModule *msg = new deleteThisModule;
     scheduleAt(simTime(), msg);
-  } else {
-    myAddress = getParentModule()->par("address");
-    EndToEndConnection = par("EndToEndConnection");
-    number_of_resources = par("NumberOfResources");
-    num_measure = par("num_measure");
-
-    Addresses_of_other_EndNodes = storeEndNodeAddresses();
-    int tp = par("TrafficPattern");
-
-    // cModule *qnode = getQNode();
-    // if(myAddress == 1 && EndToEndConnection){//hard-coded for now
-    // 20/3/15: upgrade from "exactly one connection" to
-    // "let's all mambo!"  Each EndNode makes exactly one connection.
-    // that means that some nodes will be receivers of more than
-    // one connection, at random.
-    // myaddress==1 for debugging
-    if (EndToEndConnection) {  // hard-coded for now
-      int endnode_destination_address;
-      int ia;
-
-      switch (tp) {
-        default:
-          error("Invalid TrafficPattern specified.");
-          break;
-        case 0:
-          EV << "EndToEndConnection is set. but no traffic pattern specified; proceeding with no traffic\n";
-          break;
-        case 1:  // just one connection
-          ia = par("LoneInitiatorAddress");
-          if (myAddress == ia) {
-            // while ((endnode_destination_address = getOneRandomEndNodeAddress()) == myAddress);
-            endnode_destination_address = getOneRandomEndNodeAddress();
-            EV << "Just one lonely connection setup request will be sent from " << myAddress << " to " << endnode_destination_address << "\n";
-            ConnectionSetupRequest *pk = new ConnectionSetupRequest();
-            pk->setActual_srcAddr(myAddress);
-            pk->setActual_destAddr(endnode_destination_address);
-            pk->setDestAddr(myAddress);
-            pk->setSrcAddr(myAddress);
-            pk->setNumber_of_required_Bellpairs(number_of_resources);
-            pk->setKind(7);
-            scheduleAt(simTime(), pk);
-          }
-          break;
-        case 2:  // let's all mambo!
-          // while ((endnode_destination_address = getOneRandomEndNodeAddress()) == myAddress);
-          endnode_destination_address = getOneRandomEndNodeAddress();
-          EV << "My connection setup request will be sent from " << myAddress << " to " << endnode_destination_address << "\n";
-          ConnectionSetupRequest *pk = new ConnectionSetupRequest();
-          pk->setActual_srcAddr(myAddress);
-          pk->setActual_destAddr(endnode_destination_address);
-          pk->setDestAddr(myAddress);
-          pk->setSrcAddr(myAddress);
-          pk->setNumber_of_required_Bellpairs(number_of_resources);  // required bell pairs
-          pk->setKind(7);
-          // to avoid conflict
-          scheduleAt(simTime() + exponential(0.00001 * myAddress), pk);
-          break;
-      }
+    return;
```

Comment:
we know that no more to do something in this method, so I wrote this to return.
we should keep the scope small to make it more readable.